NSS-Bashful-Turtles/Yak-Social-Network
# Rm profile auth

Filled out components on profile view with all parts per issue ticket.

to test
```
git fetch --all
git checkout rm-profile-auth
```

1. make sure json server and npm dev server are running
1. load up web page, user profile 1 is currently hard coded to load
1. if profile 1 is currently in session storage, you should see a button to change theme that is currently disabled.
1. change value in index.js to call a different profile that is not the one logged in
1. profile view should now display follow and add friend buttons
1. review code
